javascript/javascript3/week1/forecast/styles.css

```diff
@@ -0,0 +1,122 @@
+body{
+    font-family: monospace;
+    color: rgb(13, 160, 228)
+}
+.logo{
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+    justify-content: center;
+}
+#logo{
+    margin-top: 35%;
+    width: 200px;
+    height: auto;
+}
```

Comment:
In between classes, you sometimes put spaces, sometimes don't: try to be consistent.